src/main/java/com/kodilla/ecommercee/domain/Group.java

```diff
@@ -0,0 +1,63 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Entity(name = "GROUPS")
```

Comment:
Zastanawiam się, czy nie trzeba będzie nieznacznie zmienić nazwy encji. U mnie tak nazwana encja powodowała błędy składni SQL: group, groups to są słowa kluczowe dla baz danych: http://www.h2database.com/html/advanced.html#keywords

Reply:
zmieniłam na GROUP_ALL